src/Core/Migration/V6_7/Migration1745319883AddDefaultConfigForMeasurementSystem.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+/**
+ * @internal
+ */
+#[Package('inventory')]
+class Migration1745319883AddDefaultConfigForMeasurementSystem extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1745319883;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $query = 'INSERT IGNORE INTO system_config SET
+                    id = :id,
+                    configuration_value = :configValue,
+                    configuration_key = :configKey,
+                    created_at = :createdAt;';
+
+        $metricId = $connection->fetchOne('SELECT id FROM `measurement_system` WHERE `technical_name` = "metric"');
```

Comment:
what's the reason for this query? We created it two migrations before, we can expect it to be there, and likely the system is more broken when the system config entry does not exist at all as when the referenced unit system does not exist

Reply:
Additionally as those entries are only referenced by name, how do we ensure that they are updated/deleted etc?

Reply:
Makes sense

for other system config (e.g. cms pages IIRC) I think there are some "protections" that you can't delete the current page if it is referenced by the system config, but yes that has to be handled on application level as we don't have DB constraints

Maybe you can check if you find something similar and create a follow up ticket for that?